### PR TITLE
Add support for cluster tags

### PR DESCRIPTION
**Pull request by iovadia**
Signed-off-by: Ido Ovadia <iovadia@redhat.com>

### CODE REVIEW

File: skipper.env [hunk 1]
```diff
@@ -137,3 +137,4 @@ NUTANIX_CLUSTER_NAME
 NUTANIX_SUBNET_NAME
 TF_LOG_PATH
 TF_LOG
+TAGS
```
Comment (eliorerz) [inline skipper.env:140]: Rename it to `CLUSTER_TAGS`, TAGS is just to generic

File: src/assisted_test_infra/test_infra/helper_classes/config/cluster_config.py [hunk 2]
```diff
@@ -17,6 +17,7 @@ class BaseClusterConfig(BaseEntityConfig):
 
     cluster_id: str = None
     cluster_name: ClusterName = None
+    tags: str = None
```
Comment (eliorerz) [inline src/assisted_test_infra/test_infra/helper_classes/config/cluster_config.py:20]: Same here
```suggestion
    cluster_tags: str = None
```

File: src/tests/global_variables/env_variables_defaults.py [hunk 3]
```diff
@@ -19,6 +19,7 @@ class _EnvVariables(DataPool, ABC):
     openshift_version: EnvVar = EnvVar(["OPENSHIFT_VERSION"], default=consts.OpenshiftVersion.DEFAULT.value)
     base_dns_domain: EnvVar = EnvVar(["BASE_DOMAIN"], default=env_defaults.DEFAULT_BASE_DNS_DOMAIN)
     cluster_name: EnvVar = EnvVar(["CLUSTER_NAME"], default="")
+    tags: EnvVar = EnvVar(["TAGS"], default="")
```
Comment (eliorerz) [inline src/tests/global_variables/env_variables_defaults.py:22]: And here
```suggestion
    cluster_tags: EnvVar = EnvVar(["CLUSTER_TAGS"], default="")
```

**Review by eliorerz — CHANGES_REQUESTED** [review]

File: skipper.env (hunk 1 above)
Comment (omertuc) [inline skipper.env:140]: Also restore the newline at the end of the file

Comment (iovadia) [reply skipper.env]: Done

File: src/tests/global_variables/env_variables_defaults.py (hunk 3 above)
Comment (iovadia) [reply src/tests/global_variables/env_variables_defaults.py]: Done

File: src/assisted_test_infra/test_infra/helper_classes/config/cluster_config.py (hunk 2 above)
Comment (iovadia) [reply src/assisted_test_infra/test_infra/helper_classes/config/cluster_config.py]: Done

File: skipper.env (hunk 1 above)
Comment (iovadia) [reply skipper.env]: Done

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py [hunk 4]
```diff
@@ -193,6 +195,14 @@ def download_infra_env_image(self, iso_download_path=None) -> Path:
         log.debug(f"Downloading ISO to {iso_download_path}")
         return self._infra_env.download_image(iso_download_path=iso_download_path)
 
+    def update_tags(self, tags: str):
+        log.info(f"Setting cluster tags: {tags} for cluster: {self.id}")
+        self.update_config(cluster_tags=tags)
+        self.api_client.update_cluster(self.id, {"tags": tags})
+
+    def get_tags(self) -> str:
```
[the remark this answers is not in the document]
Comment (eliorerz) [reply src/assisted_test_infra/test_infra/helper_classes/cluster.py:203]: Oh.. I understand, it's a tricky one. I guess that we need to sync them.
Maybe
```suggestion
    def get_tags(self) -> str:
        tags = self.get_details().tags
        self._config.cluster_tags = tags
        return tags
```

**Review by eliorerz — APPROVED** [review]
LGTM

**Review by eliorerz — APPROVED** [review]

**Review by eliorerz — APPROVED** [review]
LGTM

**Pull request merged**